Classes/Request/YLPGeoCoordinate.m

```diff
@@ -31,15 +31,15 @@ - (instancetype)initWithLatitude:(double)latitude
 - (NSString *)description {
     NSString *baseString = [self latLongString];
     
-    if (self.accuracy) {
+    if (self.accuracy != 0) {
         baseString = [self accuracyDescription:baseString];
     }
     
-    if (self.altitude) {
+    if (self.altitude != 0) {
         baseString = [self altitudeDescription:baseString];
     }
     
-    if (self.altitudeAccuracy) {
+    if (self.altitudeAccuracy != 0) {
         baseString = [self altitudeAccuracyDescription:baseString];
```

Comment:
@d9chen, are these `!= 0` checks correct? I don't totally understand why we would exclude these values if they were 0, but that's what the previous code is equivalent to.